### PR TITLE
[13.0] [MIG] account invoice import ubl

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

File: account_invoice_import_ubl/wizard/account_invoice_import.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+        if qty_xpath:
+            if float(qty_xpath[0].text):
+                qty = float(qty_xpath[0].text)
+            if qty_xpath[0].attrib and qty_xpath[0].attrib.get("unitCode"):
+                unece_uom = qty_xpath[0].attrib["unitCode"]
+                if unece_uom == "ZZ":
+                    unece_uom = "C62"
+                uom = {"unece_code": unece_uom}
+        product_dict = self.ubl_parse_product(iline, namespaces)
+        name_xpath = iline.xpath("cac:Item/cbc:Description", namespaces=namespaces)
```
Comment (CasVissers-360ERP) [inline account_invoice_import_ubl/wizard/account_invoice_import.py:67]: Description is not oftenly filled, name is more common (product name). I would suggest something like:
```
        name_xpath = iline.xpath("cac:Item/cbc:Name", namespaces=namespaces)
        description_xpath = iline.xpath("cac:Item/cbc:Description", namespaces=namespaces)
        if name_xpath[0].text and description_xpath[0].text:
            name = name_xpath[0].text + " / " and description_xpath[0].text
        elif name_xpath[0].text and not description_xpath[0].text:
            name = name_xpath[0].text
        else:
            name = description_xpath and description_xpath[0].text or "-"

```

**Review by CasVissers-360ERP — CHANGES_REQUESTED** [review]
Only one minor comment.

File: account_invoice_import_ubl/wizard/account_invoice_import.py (hunk 1 above)
Comment (StefanRijnhart) [reply account_invoice_import_ubl/wizard/account_invoice_import.py:67]: @NL66278 what do you think of this suggestion?

Comment (NL66278) [reply account_invoice_import_ubl/wizard/account_invoice_import.py:67]: Seems a logical thing todo to me. Using the maximum amount of information from the invoice.

**Pull request closed without merging**